### PR TITLE
TRUNK-4986: Upgrade liquibase to 3.5.x version

### PR DESCRIPTION
**Pull request by georgejunior**
## Description of what I changed
I worked on TRUNK-4986: TRUNK-4986: Upgrade liquibase to 3.5.x version

## Issue I worked on
see https://issues.openmrs.org/browse/TRUNK-4986

## Checklist: I completed these to help reviewers :)

- [x ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/resources/liquibase-update-to-latest.xml [hunk 1]
```diff
@@ -20,5 +20,7 @@
     <include file="liquibase-update-to-2.1.xml"/>
     <include file="liquibase-update-to-2.2.xml"/>
     <include file="liquibase-update-to-2.3.xml"/>
+    <include file="liquibase-update-to-latest.xml"/>
+    <include file="liquibase-update-to-3.5.xml"/>
```
Comment (Ruhanga) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @georgejunior , what are these changes for?

File: api/src/main/resources/liquibase-update-to-2.3.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     If you intend for a changeset to be backported to older releases, you must put such a changeset in the
     liquibase-update-to-2.0.xml, which have been used for older releases.
 -->
-<databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+<databaseChangeLog logicalFilePath="liquibase-update-to-3.5s.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```
Comment (Ruhanga) [inline api/src/main/resources/liquibase-update-to-2.3.xml]: What is this change for?

File: api/src/main/resources/liquibase-update-to-2.0.xml [hunk 3]
```diff
@@ -10,7 +10,7 @@
     graphic logo is a trademark of OpenMRS Inc.
 
 -->
-<databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+<databaseChangeLog logicalFilePath="liquibase-update-to-3.5.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```
Comment (mozzy11) [inline api/src/main/resources/liquibase-update-to-2.0.xml]: what was this change for ?

File: api/src/main/resources/liquibase-update-to-2.1.xml [hunk 4]
```diff
@@ -15,7 +15,7 @@
     If you intend for a changeset to be backported to older releases, you must put such a changeset in the
     liquibase-update-to-2.0.xml, which have been used for older releases.
 -->
-<databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+<databaseChangeLog logicalFilePath="liquibase-update-to-3.5.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```
Comment (mozzy11) [inline api/src/main/resources/liquibase-update-to-2.1.xml]: what was this change for also ??

File: api/src/main/resources/liquibase-update-to-2.2.xml [hunk 5]
```diff
@@ -15,7 +15,7 @@
     If you intend for a changeset to be backported to older releases, you must put such a changeset in the
     liquibase-update-to-2.0.xml, which have been used for older releases.
 -->
-<databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+<databaseChangeLog logicalFilePath="liquibase-update-to-3.5.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```
Comment (mozzy11) [inline api/src/main/resources/liquibase-update-to-2.2.xml]: what was this change for ??

**Comment by mozzy11** [comment]
@georgejunior  have you seen the travis failure ?? you seem to have undesired changes

**Comment by HerbertYiga** [comment]
is  3.5.x the  latest version of liquidate-core  as indicated in your pom?

**Comment by samuelmale** [comment]
@georgejunior Did you see the Travis failure?

**Comment by georgejunior** [comment]
> @georgejunior Did you see the Travis failure?

@samuelmale i saw it

**Comment by mozzy11** [comment]
@georgejunior  can you squash your commits into one as advised here https://wiki.openmrs.org/display/docs/Pull+Request+Tips.

* First Remove all other changes and only do a version upgrade for now.

**Comment by georgejunior** [comment]
i am closing this one because they are two pull requests

**Pull request closed without merging**